spec/models/child_theme_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe ChildTheme do
```

Comment:
It looks like we're missing a test case for the `I18n.t("themes.errors.no_multilevels_components")` error